IFTPipeline.jl/test/test-feature-extract.jl

```diff
@@ -25,15 +25,27 @@ end
 @testset "feature-extraction.jl" begin
     image_size = (8, 8)
```

Comment:
Why not make this as small as possible?
```suggestion
    image_size = (1, 1)
```

Reply:
I want to have it select a range of values – it gives us a bit more security than just a single value each time (might get negatives and positives for signed types, for instance) and the computational cost is quite small.

Reply:
I see your point but isn't the goal testing type casting? One single random value is as random as any other 63 of them. You might not get any negative or positive values. If you'd like to test for that consider ensuring the full spectrum will be represented in the samples.

Reply:
Part of it is type casting, yes. But we're also checking that the image doesn't get inverted, or mirrored or something else.
I agree – ideally we'd use something like https://quickcheckjl.readthedocs.io/en/latest/, or test the minimum value of the datatype, maximal value, zeros, repeats etc. to make this checking more robust.
At the very least I want it to test a range of values, and not just a single value. 8x8 seems a reasonable balance of speed/memory/effort vs robustness of testing at this point.